### PR TITLE
Feature: Inline background image

### PR DESCRIPTION
**Pull request by tomusborne**
This adds an inline background image option to our Container block.

### CODE REVIEW

File: src/blocks/element/components/InlineBackgroundImage.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+		if ( ! inlineBackgroundPart ) {
+			return '';
+		}
+
+		return inlineBackgroundPart.split( 'url(' )[ 1 ].split( ')' )[ 0 ];
+	}, [ htmlAttributes?.style ] );
+
+	function onChange( value ) {
+		const style = htmlAttributes?.style
```
Comment (iansvo) [inline src/blocks/element/components/InlineBackgroundImage.jsx:29]: Seems like you could use destructuring here to do the same thing but a bit more readable?

```
const { style = '' } = htmlAttributes;
```

File: src/blocks/element/utils.js [hunk 2]
```diff
@@ -1,10 +1,22 @@
 export const convertInlineStyleStringToObject = ( styleString ) => {
```
Comment (iansvo) [inline src/blocks/element/utils.js:1]: One thought here....if we are talking about inline styles and we have a selected block (via useSelectedBlockElement) then we could just get the styles attribute off the element directly and not have to build the object at all.

**Review by iansvo — COMMENTED** [review]
Couple of feedback items, no hard required chagnes though.

File: src/blocks/element/utils.js (hunk 2 above)
Comment (tomusborne) [reply src/blocks/element/utils.js:1]: Would be simpler, but we need this to work when the block isn't selected as well.

I tried to do a `ref.current.style`, but that gave me a ton of things we don't need.

We can improve this in a future PR if we have a better way.

**Pull request merged**